exercises/concept/annalyns-infiltration/.docs/introduction.md

```diff
@@ -0,0 +1,56 @@
+# Introduction
+
+## Booleans
+
+As in most other programming languages, a Boolean type in Rust has two possible values: `true` and `false`.
+
+### Declaration
+The Boolean type in Rust is specified using `bool`.
+
+```rust
+let t = true;
+
+let f: bool = false; // with explicit type annotation
+```
+
+The main way to use Boolean values is through conditionals, such as an if
```

Comment:
There is even more duplication here. The text explaining booleans seems to be copied three times? in the concept's about, the concept's introduction and here in the exercise's introduction.

Reply:
This file is documented here: https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-introduction-md